labellab-client/src/components/project/css/images.css

```diff
@@ -47,3 +47,8 @@
   flex: 1;
   outline: 0;
 }
+@media only screen and (max-width: 902px){
+  .image-table-body{
+    width: 902px;
```

Comment:
Restricting the width constant dimension is not a good practice.

Reply:
@Tushar19varshney I know but without using it the card shrinks in size and the top bar of that table spills out. I tested it and it works fine.